evalml/tests/dependency_update_check/latest_dependency_versions.txt

```diff
@@ -6,7 +6,7 @@ featuretools==0.23.0
 graphviz==0.16
 ipywidgets==7.6.3
 kaleido==0.1.0
-lightgbm==3.1.1
```

Comment:
I don't think we need to update this? This just tracks the latest published version, not the latest version allowed in our requirements.

Reply:
Ohh did not know that. I will change this, thanks!